# Snapshot report for `test/test_opt.js`

The actual snapshot is saved in `test_opt.js.snap`.

Generated by [AVA](https://ava.li).

## [ava] [multipleResolves] should make tests fails

> Snapshot 1

    {
      code: 1,
      stderr: '',
      stdout: `␊
        Uncaught exception in build/test/ava.js␊
      ␊
      ␊
        Error:  ✖  multipleResolves (a promise was resolved/rejected multiple times)  Initially resolved with: { success: true }␊
            Then rejected with: Error: message␊
                at STACK TRACE␊
      ␊
        ✔ should make tests fail on multipleResolves␊
        ✖ build/test/ava.js exited with a non-zero exit code: 1␊
      ␊
        1 test passed␊
        1 uncaught exception␊
      ␊
      `,
    }

## [ava] [rejectionHandled] should make tests fails

> Snapshot 1

    {
      code: 1,
      stderr: '',
      stdout: `␊
        Uncaught exception in build/test/ava.js␊
      ␊
      ␊
        Error:  ✖  rejectionHandled (a promise was rejected and handled too late)  Error: message␊
                at STACK TRACE␊
      ␊
        ✔ should make tests fail on rejectionHandled␊
        ✖ build/test/ava.js exited with a non-zero exit code: 1␊
      ␊
        1 test passed␊
        1 uncaught exception␊
      ␊
      `,
    }

## [ava] [uncaughtException] should make tests fails

> Snapshot 1

    {
      code: 1,
      stderr: '',
      stdout: `␊
        Uncaught exception in build/test/ava.js␊
      ␊
      ␊
        Error: message␊
      ␊
        ✖ build/test/ava.js exited with a non-zero exit code: 1␊
      ␊
        1 uncaught exception␊
      ␊
      `,
    }

## [ava] [unhandledRejection] should make tests fails

> Snapshot 1

    {
      code: 1,
      stderr: '',
      stdout: `␊
        ✔ should make tests fail on unhandledRejection␊
      ␊
        Unhandled rejection in build/test/ava.js␊
      ␊
      ␊
        Error: message␊
      ␊
      ␊
        1 test passed␊
        1 unhandled rejection␊
      ␊
      `,
    }

## [ava] [warning] should make tests fails

> Snapshot 1

    {
      code: 1,
      stderr: '',
      stdout: `␊
        Uncaught exception in build/test/ava.js␊
      ␊
      ␊
        Error:  ✖  warning  (500) Detail␊
            Error: message␊
                at STACK TRACE␊
      ␊
        ✔ should make tests fail on warning␊
        ✖ build/test/ava.js exited with a non-zero exit code: 1␊
      ␊
        1 test passed␊
        1 uncaught exception␊
      ␊
      `,
    }
